.github/workflows/destroy/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Destroy PAAS environment
```

Comment:
As discussed, we can remove this action and use the make commands instead if necessary